README.md

```diff
@@ -47,14 +48,14 @@ releases use the fixed tag of the release.
 To select a specific Docker image tag set `DOCKER_TAG` when installing
 `vsyncer`:
 
-    make install PREFIX=/path/to/bin DOCKER_TAG=v2.0.0
+    make install PREFIX=/path/to/bin DOCKER_TAG=latest
 
 ### Disabling Docker
 
 To run `vsyncer` without Docker, you'll need the following tools:
 
 - clang and llvm >= v14
-- Dartagnan >= v4.0.0 (alternative)
+- Dartagnan >= v4.0.1 (alternative)
```

Comment:
I would prefer not to have this until we actually make the release. I want to merge just one more PR from Thomas, ask you to test it in the demo and then I will release.